web/data/perCountryDataCaseCounts.json

```diff
@@ -30,7 +30,8 @@
         "22C (Omicron)",
         "22D (Omicron)",
         "S:677H.Robin1",
-        "S:677P.Pelican"
+        "S:677P.Pelican",
+        "others"
       ],
```

Comment:
"others" needs to be added into the list of clusters in json data. Here I illustrate what the result should look like for cases data. This is a manual addition and is not enforced in code. Python scripts will need to be changed for that.